mpas_analysis/sea_ice/time_series.py

```diff
@@ -389,11 +385,14 @@ def run_task(self):  # {{{
                 inFilesPreprocessed = '{}/icevol.{}.year*.nc'.format(
                     preprocessedReferenceDirectory,
                     preprocessedReferenceRunName)
-                dsPreprocessed = open_multifile_dataset(
-                    fileNames=inFilesPreprocessed,
-                    calendar=calendar,
-                    config=config,
-                    timeVariableName='xtime')
```

Comment:
This bit of code did *exactly* the same thing as the block above, I think the block above was a mistake (and didn't accomplish anything).